fastlane/spec/actions_specs/setup_ci_spec.rb

```diff
@@ -1,24 +1,33 @@
 describe Fastlane do
   describe Fastlane::Actions::SetupCiAction do
+    is_macos_env = RUBY_PLATFORM.downcase.include?("darwin")
```

Comment:
I think we could probably rework this so the tests aren't dependent on the platform they are being run on 😇 

The following tests should be reworked like 👇 
```rb
describe "calls setup_keychain after setup_output_paths if :provider is set to circleci on Mac Agents" do
  it "on Mac" do
    allow(FastlaneCore::Helper).to receive(:mac?).and_return(true)

    # Message is asserted in reverse order, hence output of setup_output_paths is expected last
    expect(Fastlane::UI).to receive(:message).with("Enabling match readonly mode.")
    expect(Fastlane::UI).to receive(:message).with("Creating temporary keychain: \"fastlane_tmp_keychain\".")
    expect(Fastlane::UI).to receive(:message).with("Skipping Log Path setup as FL_OUTPUT_DIR is unset")
  end

  it "on linux" do
    allow(FastlaneCore::Helper).to receive(:mac?).and_return(false)

    expect(Fastlane::UI).to receive(:message).with("Skipping Log Path setup as FL_OUTPUT_DIR is unset")
    expect(Fastlane::UI).to receive(:message).with("Skipping Keychain setup on non-macOS CI Agent")
  end
end
```